roles/container-engine/containerd/tasks/containerd_repo.yml

```diff
@@ -24,16 +24,6 @@
     - ansible_os_family in ['Ubuntu', 'Debian']
     - containerd_repo_info.repos|length > 0
 
-# This is required to ensure any apt upgrade will not break kubernetes
```

Comment:
See commit message
```
containerd: remove duplicate package pining task
Leave it with the install instead of the repo config
```
We need to apply it each time we update so it make more sense for it to be next to the `package` task

Reply:
I see. We have the same task in container-engine/containerd/tasks/main.yml